File: airline_app/lib/models/booking.dart
class Booking {
  final int bookingId;
  final int flightId;
  final String seatNumber;
  final String passengerName;
  final String status;

  Booking({
    required this.bookingId,
    required this.flightId,
    required this.seatNumber,
    required this.passengerName,
    required this.status,
  });

  factory Booking.fromJson(Map<String, dynamic> json) {
    return Booking(
      bookingId: json['bookingId'],
      flightId: json['flightId'],
      seatNumber: json['seatNumber'],
      passengerName: json['passengerName'],
      status: json['status'],
    );
  }
}

File: airline/src/main/java/com/airline/booking/domain/Booking.java
package com.airline.booking.domain;

import jakarta.persistence.*;
import lombok.*;

@Entity
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Builder
public class Booking {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    private Long flightId;

    private String seatNumber;

    private String passengerName;

    @Enumerated(EnumType.STRING)
    private BookingStatus status;
}
